lib/vdsm/virt/vmdevices/storage.py

```diff
@@ -940,6 +940,26 @@ def on_block_threshold(self, reported_path):
             self.exceeded_time = time.monotonic_time()
             self.log.info("drive %r threshold exceeded", self.name)
 
+    def on_enospc(self):
+        """
+        Called when a VM pause because of ENOSPC writing to this drive.
```

Comment:
"pause" -> "pauses"
"because of ENOSPC writing" -> "because of ENOSPC when writing"